doc/nrf/releases/release-notes-2.3.0.rst

```diff
@@ -0,0 +1,1034 @@
+    The :ref:`modem_shell_application` sample now demonstrates how to store and upload modem trace to cloud.
+  * New :ref:`Serial LTE Modem (SLM) Shell <slm_shell_sample>` sample:
+    The sample demonstrates sending AT commands to the nRF9160 SiP from shell for nRF53 and nRF52 Series SoCs.
+  * New :ref:`mqtt_sample` sample supporting Wi-Fi together with cellular connectivity (replacing the Simple MQTT sample).
+
+* Added :ref:`experimental support <software_maturity>` for the following features:
+
+  * Periodic Advertisement with Responses (PAwR) - Advertiser.
+    This is a new feature introduced in Bluetooth v5.4 specification, enabling the bidirectional exchange of application data using connectionless communication.
+    Learn more in our `Bluetooth 5.4 DevZone blog`_.
+    The SoftDevice Controller continues to be Bluetooth v5.3 qualified.
+  * Bluetooth LE Audio: Public Broadcast Announcement (PBA) and bidirectional streams to and from two headsets.
+  * New :ref:`wifi_sr_coex_sample` sample.
+
+* Improved:
+
+  * The :ref:`matter_lock_sample` sample has been extended to support switching between Matter over Thread and Matter over Wi-Fi during application operation.
+  * The :ref:`wifi_shell_sample` sample by adding the Power Save feature.
+
+See :ref:`ncs_release_notes_230_changelog` for the complete list of changes.
+
+Sign up for the `nRF Connect SDK v2.3.0 webinar`_ to learn more about the new features.
+
+The official nRF Connect for VS Code extension has also been improved.
+See the `latest release notes for nRF Connect for Visual Studio Code`_ for more information.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.3.0**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.3.0`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the only officially supported IDE for |NCS| v2.3.0.
+SEGGER Embedded Studio Nordic Edition is no longer tested or expected to work with |NCS| v2.3.0.
+
+:ref:`Toolchain Manager <gs_app_tcm>`, used to install the |NCS| automatically from `nRF Connect for Desktop`_, is available for Windows, Linux, and macOS.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
```

Comment:
We usually avoid to have headers without a following description, so a short desc to introduce the table after this header would be in order. But do we even need this header? Isn't "Supported modem firmware" covering the table already?

Reply:
I will leave it as this serves as a title for the table. We do have a ton of section headings without short descriptions in this document.